src/odemis/gui/win/acquisition.py

```diff
@@ -780,18 +789,36 @@ def __init__(self, parent, orig_tab_data):
         # imprecision, all the tiles will overlap or at worse be next to each other (i.e. , no space between tiles)
         self.overlap = 0.10
         try:
+            if self.fibsem_mode:
+                self.stage = self._main_data_model.stage_bare
+                self.focuser = self._main_data_model.ebeam_focus
+                self.detector = self._main_data_model.sed
+                self.settings_obs = None
+                range_md = model.MD_SEM_IMAGING_RANGE
+
+                # In FIBSEM mode, we don't have autofocus because of how the overview code currently works
+                self.autofocus_chkbox.Hide()
+                self.autofocus_roi_ckbox.value = False
+
+            else:
+                self.stage = self._main_data_model.stage
+                self.focuser = self._main_data_model.focus
+                self.detector = self._main_data_model.ccd
+                self.settings_obs = self._main_data_model.settings_obs
+                range_md = model.MD_POS_ACTIVE_RANGE
+
             # Use the stage range, which can be overridden by the MD_POS_ACTIVE_RANGE.
             # Note: this last one might be temporary, until we have a RoA tool provided in the GUI.
             self._tiling_rng = {
-                "x": self._main_data_model.stage.axes["x"].range,
-                "y": self._main_data_model.stage.axes["y"].range
+                "x": self.stage.axes["x"].range,
+                "y": self.stage.axes["y"].range
             }
 
-            stage_md = self._main_data_model.stage.getMetadata()
-            if model.MD_POS_ACTIVE_RANGE in stage_md:
-                self._tiling_rng.update(stage_md[model.MD_POS_ACTIVE_RANGE])
+            stage_md = self.stage.getMetadata()
+            if range_md in stage_md:
```

Comment:
I cannot really think of a better name for `range_md`, but it reads a bit weird that `stage_md` contains all the metadata, while `range_md` is only a single metadata entry. So if you think of a better name, can you update it?

Reply:
changed to imaging_range, as its an an artificical limit on the stage for imaging